pkg/platformclient/release.go

```diff
@@ -84,3 +85,7 @@ func (c *HTTPClient) PromoteRelease(appID string, sequence int64, label, notes s
 	}
 	return nil
 }
+
+func (c *HTTPClient) LintRelease(appID string, yaml string) ([]types.LintMessage, error) {
+	return nil, nil
```

Comment:
seems appropriate to return the error here

Reply:
done